apps/spinner/src/chance-wheel.jsx

```diff
@@ -0,0 +1,223 @@
+import React, { useEffect , useState } from "zarkit/react";
+import "./chance-wheel.scss";
+
+
+
+function ChanceWheel (props){
+
+  const [state, setState] = useState({
```

Comment:
some states seems not required and could be removed.
like if you do calculate the result and the time the spinner will spin you wont need a boolean to check if it is!
these is nothing wrong with that idea but it seems its extra.